### DISCUSSION TITLE
Income, Quality of Life

### DISCUSSION THREAD

**Discussion by Silvenga** [actualbudget/releases#162] [Feature ideas]
One of the features I miss from YNAB is the automatic jumping from the category to either the `Deposit` or `Payment` field based on the category selected. 

In Actual, `Income` is a special category. I propose, if the category is within `Income`, then the next field selected should be `Deposit` by default.

Moved from #159, because "back in my day, we didn't have discussions, we only had issues"

**Comment by bpaulien** [comment]
And in mobile, instead of defaulting to - numbers, if an income category is selected, the negative could be automatically removed. I just noticed a pending income in my checking account so put in the transaction for tomorrow's date, but when I went back to the budget I was overbudgeted by that amount. Then I had to go back and edit the transaction and hit the negative button to switch it to positive.
